man/udevadm.xml

```diff
@@ -363,7 +363,9 @@
           <term><option>-e</option></term>
           <term><option>--exit</option></term>
           <listitem>
-            <para>Signal and wait for systemd-udevd to exit.</para>
+            <para>Signal and wait for systemd-udevd to exit. Note that systemd-udevd.service contains
+            <option>Restart=always</option> and so as a result, this option restarts systemd-udevd.
+            If you want to stop systemd-udevd.service, please use <command>systemctl</command>.</para>
```

Comment:
maybe even say "systemctl stop" here, instead of just "systemctl"?

Reply:
hmm, actually maybe even say the full commandline, since it is not obvious:

```
systemctl stop systemd-udevd-control.socket systemd-udevd-kernel.socket systemd-udevd.service
```